packages/frontend-2/pages/server-management/projects.vue

```diff
@@ -0,0 +1,256 @@
+        </div>
+      </template>
+
+      <template #created="{ item }">
+        <div class="font-mono text-xs">
+          {{ new Date(item.createdAt).toLocaleString('en-GB') }}
+        </div>
+      </template>
+
+      <template #modified="{ item }">
+        <div class="font-mono text-xs">
+          {{ new Date(item.updatedAt).toLocaleString('en-GB') }}
+        </div>
+      </template>
+
+      <template #models="{ item }">
+        <div class="font-mono text-xs">
+          {{ item.models.totalCount }}
+        </div>
+      </template>
+
+      <template #versions="{ item }">
+        <div class="font-mono text-xs">
+          {{ item.versions.totalCount }}
+        </div>
+      </template>
+
+      <template #contributors="{ item }">
+        <Avatar
+          v-for="(teamMember, index) in item.team"
+          :key="index"
+          :size="small"
+          :user="item.user"
+          class="-mr-2"
+        />
+      </template>
+    </Table>
+
+    <InfiniteLoading
+      :settings="{ identifier: infiniteLoaderId }"
+      class="py-4"
+      @infinite="infiniteLoad"
+    />
+
+    <ProjectDeleteDialog
+      v-model:open="showProjectDeleteDialog"
+      :project="projectToModify"
+      title="Delete Project"
+      :buttons="[
+        {
+          text: 'Delete',
+          props: { color: 'danger', fullWidth: true },
+          onClick: deleteConfirmed
+        },
+        {
+          text: 'Cancel',
+          props: { color: 'secondary', fullWidth: true, outline: true },
+          onClick: closeProjectDeleteDialog
+        }
+      ]"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue'
+import { useQuery } from '@vue/apollo-composable'
+import { debounce } from 'lodash-es'
+
+import Table from '~~/components/server-management/Table.vue'
+import ProjectDeleteDialog from '~~/components/server-management/DeleteProjectDialog.vue'
+import Avatar from '~~/components/user/Avatar.vue'
+import { Project } from '~~/lib/common/generated/gql/graphql'
+
+import { InfiniteLoaderState } from '~~/lib/global/helpers/components'
+import { graphql } from '~~/lib/common/generated/gql'
+
+import { MagnifyingGlassIcon, TrashIcon } from '@heroicons/vue/20/solid'
+
+definePageMeta({
+  middleware: ['admin']
+})
+
+const projectToModify = ref<Project | null>(null)
+const searchString = ref('')
+const showProjectDeleteDialog = ref(false)
+
+const openProjectDeleteDialog = (project: Project) => {
+  projectToModify.value = project
+  showProjectDeleteDialog.value = true
+}
+
+const closeProjectDeleteDialog = () => {
+  showProjectDeleteDialog.value = false
+}
+
+const handleSearchChange = (newSearchString: string) => {
+  searchUpdateHandler(newSearchString)
+}
+
+const router = useRouter()
+
+const handleProjectClick = (project: Project) => {
+  router.push(`/projects/${project.id}`)
+}
+
+const getProjects = graphql(`
+  query Query($query: String, $orderBy: String, $limit: Int!, $visibility: String) {
```

Comment:
vague query name

Reply:
Updated to getAdminProjectsQuery

Reply:
Operation names should always be in PascalCase
Class/Type names should always be in PascalCase
Component props in templates should always be in kebab-case
Component names in templates should always be in PascalCase
Functions/constants/variables should always be in camelCase

Very rarely global constants can be in uppercase SNAKE_CASE